tests/test_vor2vel.f90

```diff
@@ -82,7 +82,7 @@ program test_vor2vel
         l = twopi
         k = two * l
 
-        k2l2 = k ** 2 - l ** 2
+        klsq = k ** 2 - l ** 2
```

Comment:
This is supposed to be k^2 - l^2, right, not k^2 + l^2?

Reply:
I can currently not remember the exact setup/equations here ... I think it is correct the way it is right now. It is just a bad naming.